src/sentry/static/sentry/app/components/sparklines/index.tsx

```diff
@@ -0,0 +1,29 @@
+import {Sparklines} from 'react-sparklines';
+import PropTypes from 'prop-types';
+
+/**
+ * This is required because:
+ *
+ * - React.Suspense only works with default exports
+ * - typescript complains that the library's `propTypes` does not
+ * have `children defined.
+ * - typescript also won't let us access `Sparklines.propTypes`
+ */
+class LolSparklines extends Sparklines {
```

Comment:
Lol?

Reply:
```suggestion
class FooSparklines extends Sparklines {
```

maybe `FooSparklines` ✨